src/wallet/rpcwallet.cpp

```diff
@@ -1359,6 +1359,24 @@ static void ListTransactions(interfaces::Chain::Lock& locked_chain, CWallet* con
     }
 }
 
+static const std::string TransactionDescriptionString()
+{
+    return "    \"confirmations\": n,                        (numeric) The number of confirmations for the transaction. Negative confirmations indicate the\n"
+           "                                                       transaction conflicts with the block chain.\n"
+           "    \"generated\": xxx,                          (bool) Only returns true if transaction only input is a coinbase one, otherwise false.\n"
+           "    \"trusted\": xxx,                            (bool) Whether we consider the outputs of this unconfirmed transaction safe to spend.\n"
+           "    \"blockhash\": \"hashvalue\",                (string) The block hash containing the transaction.\n"
+           "    \"blockindex\": n,                           (numeric) The index of the transaction in the block that includes it.\n"
+           "    \"blocktime\": xxx,                          (numeric) The block time in seconds since epoch (1 Jan 1970 GMT).\n"
+           "    \"txid\": \"transactionid\",                 (string) The transaction id.\n"
+           "    \"walletconflicts: \"hashvalue\",            (array) List of txids that conflict with this transaction.\n"
```

Comment:
It returns array here. So `\"walletconflicts\": [ \"TXID\", ... ]` should be used instead. See #16869 as an example.